backend/django/core/views/api_admin.py

```diff
@@ -323,3 +325,44 @@ def heat_map_data(request, project_pk):
         coders.append({"name": str(p.profile), "pk": p.profile.pk})
 
     return Response({"data": heatmap_data, "labels": labels, "coders": coders})
+
+
+@api_view(["GET"])
+@permission_classes((IsAdminOrCreator,))
+def get_project_status(request, project_pk):
+    """This returns the labeled data.
+
+    Args:
+        request: The GET request
+        project_pk: Primary key of the project
+    Returns:
+        data: a list of data information
+    """
+
+    project = Project.objects.get(pk=project_pk)
+
+    total_data_objs = list(project.data_set.filter(datalabel__isnull=True))
```

Comment:
So the total here is actually going to be too low, since you're filtering out the labeled data. 

The "other-labeled" value is "total" - "final" - "unlabeled", so "total" should be just the length of `project.data_set`

To test this, check out the three person default project. It's got a negative number right now for "other"